src/pages/accounting/index.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles, withStyles, Theme, createStyles } from '@material-ui/core/styles';
+import { Tabs, Tab, AppBar, Box, Typography } from '@material-ui/core';
+
+import { beeDebugApi } from '../../services/bee';
+import AccountCard from '../accounting/AccountCard';
+import BalancesTable from './BalancesTable';
+import ChequebookTable from './ChequebookTable';
+import SettlementsTable from './SettlementsTable';
+import EthereumAddressCard from '../status/EthereumAddressCard';
+
+interface TabPanelProps {
+    children?: React.ReactNode;
+    index: any;
+    value: any;
+}
+
+
+function a11yProps(index: any) {
+    return {
+        id: `simple-tab-${index}`,
+        'aria-controls': `simple-tabpanel-${index}`,
+    };
+}
+
+export default function Accounting() {
+    const [value, setValue] = React.useState(0);
+
+    const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+        setValue(newValue);
+    };
+
+    const [chequebookAddress, setChequebookAddress] = useState({ chequebookaddress: '' });
+    const [loadingChequebookAddress, setLoadingChequebookAddress] = useState(false);
+
+    const [chequebookBalance, setChequebookBalance] = useState({ totalBalance: 0, availableBalance: 0});
+    const [loadingChequebookBalance, setLoadingChequebookBalance] = useState(false);
+
+    const [peerBalances, setPeerBalances] = useState({ balances: [{peer: '-', balance: 0 }] });
+    const [loadingPeerBalances, setLoadingPeerBalances] = useState(false);
+
+    const [peerCheques, setPeerCheques] = useState({ lastcheques: [{peer: '-', lastsent: {beneficiary: '', chequebook: '', payout: 0}, lastreceived: {beneficiary: '', chequebook: '', payout: 0} }] });
+    const [loadingPeerCheques, setLoadingPeerCheques] = useState(false);
+
+    const [nodeAddresses, setNodeAddresses] = useState({ overlay: '', underlay: [""], ethereum: '', public_key: '', pss_public_key: ''});
+    const [loadingNodeAddresses, setLoadingNodeAddresses] = useState(false);
+
+    const [nodeSettlements, setNodeSettlements] = useState({ totalreceived: 0, totalsent: 0, settlements: [{peer: '-', received: 0, sent: 0}] });
+    const [loadingNodeSettlements, setLoadingNodeSettlements] = useState(false);
```

Comment:
These should probably be typed just to avoid type mistakes. E.g. one of these

```
import type { PeerBalance, BalanceResponse } from '@ethersphere/bee-js'

const [peerBalances, setPeerBalances] = useState<BalanceResponse>({ balances: [{peer: '-', balance: 0 }] });

// OR

const [peerBalances, setPeerBalances] = useState<PeerBalance[]>([{peer: '-', balance: 0 }]);
```
Btw it also likely can be just an empty array :)

Reply:
Yeah now that I know that beeJS has all the types included I agree. I dont have a ton  of experience with Typescript so I wasnt really sure how to handle this situation. Theres defintely a few places where I used types of "any" that'll need to be fixed.